programs/marinade-referral/src/instructions/deposit_sol.rs

```diff
@@ -29,7 +28,7 @@ pub struct Deposit<'info> {
     pub system_program: AccountInfo<'info>,
```

Comment:
Note to self: add comment that system_program and token_program are checked by Marinade core in the cpi call